census_view.cpp

```diff
@@ -2458,6 +2454,21 @@ void CensusDVCView::UponUpdateSingleSelection(wxUpdateUIEvent& e)
     e.Enable(is_single_sel);
 }
 
+void CensusGridView::UponUpdateSingleSelection(wxUpdateUIEvent& e)
+{
+    // We consider that in absence of any selected rows, the current row is the
+    // selected/active one, so what we actually check for here is that we do
+    // not have more than a single row selected.
+    auto const sel_blocks = grid_window_->GetSelectedRowBlocks();
+    bool const is_single_sel
+        =  sel_blocks.empty()
+        || (sel_blocks.size() == 1
+            && sel_blocks[0].GetTopRow() == sel_blocks[0].GetBottomRow())
```

Comment:
Should we also check for the current row?
```suggestion
            && sel_blocks[0].GetTopRow() == sel_blocks[0].GetBottomRow()
            && sel_blocks[0].GetTopRow() == grid_window_->GetGridCursorRow())
```